packages/flutter/test/material/filter_chip_test.dart

```diff
@@ -738,8 +736,7 @@ void main() {
     await pumpCheckmarkChip(
       tester,
       chip: selectedFilterChip(),
-      brightness: theme.brightness,
-      useMaterial3: theme.useMaterial3,
+      theme: theme,
```

Comment:
```suggestion
      brightness: theme.brightness,
      useMaterial3: theme.useMaterial3,
```

Reply:
@TahaTesser  Part of the changes in this PR change the utility methods in the test following your recommendation on a previous review, where the 2 parameters are collpased into a single ThemeData. There was a recent PR from you which took this same aproach.
Should we revert and keep the 2 parameters or use the single ThemeData param?

![image](https://github.com/flutter/flutter/assets/22084723/423cc0a5-c745-44a4-9400-4c0d2503779d)

Reply:
If you can link the PR. 

I merged two chips PRs recently and we decided to keep `brightness` and `useMaterial3`, this is from that one of those PRs. Please restore the changes.

Reply:
I just found the PR, my bad. This is fine.

https://github.com/flutter/flutter/pull/140964/files#diff-771d4e403370d0f93ff8abb7aeb2468ab94a86d69f3ccda521b89948bdbf7d14R80

Reply:
I misunderstood and thought we did the opposite.

Reply:
Yes, that's the one I was referring to. Great then